lib/generation/generators/value_objects/generation_configuration/pb_generation_configuration.dart

```diff
@@ -224,4 +228,32 @@ abstract class GenerationConfiguration with PBPlatformOrientationGeneration {
     });
     return imports;
   }
+
+  /// This method takes a command and checks if one of its child screens
+  /// is the home screen for the project, if so modify main
+  /// to redirect to the proper builder
+  bool setMainForPlatform(var newCommand, String screenName) {
+    var platformOrientationMap =
+        poLinker.getPlatformOrientationData(screenName);
+
+    platformOrientationMap.forEach((key, map) {
+      map.forEach((key, tree) {
+        if (tree.rootNode is InheritedScaffold &&
+            (tree.rootNode as InheritedScaffold).isHomeScreen) {
+          fileStructureStrategy.commandCreated(EntryFileCommand(
+              entryScreenName: (newCommand as WriteScreenCommand)
```

Comment:
If we expect `newCommand` to be of type `WriteScreenCommand`, we could instead state that in the method signature. 

Essentially replacing `var newCommand` for `WriteScreenCommand newCommand`